config/action.d/allow-iptables-multiport.conf

```diff
@@ -0,0 +1,59 @@
+# Fail2Ban configuration file for allowing hosts
+#
+# WARNING
+# Please be aware that all users behind NAT will access the service on the specified port.
+# You should protect this service with another jail that has very long bantime.
+
+[INCLUDES]
+
+before = iptables-common.conf
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = iptables -N f2b-<name>
+              iptables -A f2b-<name> -j <blocktype>
```

Comment:
What about instead of redefining many/every iptables-* action we generalize existing ones, i.e. add "returntype" variable defaulting to "RETURN" in -commons and adjusting all iptables-*conf to use it.

Then for 'allow' action it would just be needed to provide (for the pass2allow jail) for the iptables-* action of choice "blocktype=RETURN" (btw, why not ACCEPT?) and "returntype=REJECT" ?